bootstrap.d/media-libs.yml

```diff
@@ -876,6 +876,49 @@ packages:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
 
+  - name: vulkan-loader
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: Vulkan Installable Client Driver (ICD) Loader
+      description: This package provides the Vulkan Installable Client Driver (ICD) Loader. This allows the vulkan driver to be chosen at run time, rather than requiring static configuration.
+      spdx: 'Apache-2.0'
+      website: 'https://github.com/KhronosGroup/Vulkan-Loader'
+      maintainer: "Alexander Richards <electrodeyt@gmail.com>"
+      categories: ['media-libs']
+    source:
+      subdir: 'ports'
+      git: 'https://github.com/KhronosGroup/Vulkan-Loader.git'
+      tag: 'v1.3.268'
+      version: '1.3.268'
+    tools_required:
+      - system-gcc
+      - host-cmake
```

Comment:
Potentially missing dependency on wayland-scanner?

Reply:
AFAICT wayland-scanner is not needed here.